scripts/security/windows_pen_testing/Run-OpenFddWebAppSecurityScan-v3.4.ps1

```
@@ -0,0 +1,730 @@
+function Get-ZapAlertsFromJson {
+    param([string]$JsonPath)
+    $Alerts = New-Object System.Collections.Generic.List[object]
+    if (-not (Test-Path $JsonPath)) { return $Alerts }
+    try {
+        $Json = Get-Content -Raw -Path $JsonPath | ConvertFrom-Json
+        if ($Json.site) {
+            foreach ($Site in @($Json.site)) {
+                foreach ($Alert in @($Site.alerts)) { $Alerts.Add($Alert) | Out-Null }
+            }
+        }
+    } catch { }
```

PSScriptAnalyzer (1.25.0)

[warning] Empty catch block is used. Please use Write-Error or throw statements in catch blocks.

(PSAvoidUsingEmptyCatchBlock)